src/test/java/io/confluent/connect/elasticsearch/MappingTest.java

```diff
@@ -38,6 +38,7 @@
 import static org.mockito.Mockito.mock;
 import static org.mockito.Mockito.when;
 
+@SuppressWarnings("deprecation")
```

Comment:
why is this needed?

Reply:
ObjectNode is getting deprecated so the build fails with some warnings.